[New Term Entry] Java Math Methods: `.rint()` (#1006)
Codecademy/docs: content/java/concepts/math-methods/terms/rint/rint.md

---
Title: '.rint()'

> Description: 'Returns the closest whole double value to the value of the argument.'

Reviewer: ```suggestion
Description: 'Returns a double that is rounded to the closest whole integer.'
```

> Description: 'Returns the closest double value depending on the value of the argument.'

Reviewer 2: ```suggestion
Description: 'Returns the closest whole double value to the value of the argument.'
```

Subjects:
  - 'Computer Science'
Tags:

>   - 'Functions'
>   - 'Methods'
>   - 'Arithmetic'

Reviewer: ```suggestion
  - 'Functions'
  - 'Methods'
  - 'Arithmetic'
  - 'Numbers'
```

CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Math.rint()`** returns the closest whole double value to the value of the argument.

Reviewer: ```suggestion
The **`Math.rint()`** returns a double that is rounded to the closest whole integer.
```

> The **`Math.rint()`** returns the closest double value depending on the value of the argument.

Reviewer 2: ```suggestion
The **`Math.rint()`** returns the closest whole double value to the value of the argument.
```

## Syntax

```pseudo
Math.rint(num)
```

> The `.rint()` method takes a double as an argument and returns a double.

Reviewer: ```suggestion
The `.rint()` method takes a double `num` parameter and returns a double.
```

In the context of describing the function/method definition in the "Syntax" section, let's use "parameter". Any other times when the function/method is used in an actual example, we should use "argument."

> The return type of the `.rint()` method and data type of the `num` parameter is a double.

Reviewer 2: This sentence is a little clumsy. You could rewrite it as something like: "The `.rint()` method takes a double as an argument and returns a double."

Author: @SSwiniarski, done.

Some edge cases for the `.rint()` method include the following:

> - If the value of `num` is equal to an integer, then the result of the method is the value of `num`.

Reviewer: ```suggestion
- If the value of `num` is passed an integer, then it is returned as a double.
```

> - If the value of `num` is `NaN` or infinity, then the result of the method is the value of `num`.

Reviewer: ```suggestion
- If the value of `num` is `NaN` or positive/negative infinity, then the value of `num` is returned.
```

> - If the value of `num` is NaN or infinity, then the result of the method is the value of `num`.

Reviewer 2: ```suggestion
- If the value of `num` is `NaN` or infinity, then the result of the method is the value of `num`.
```

> - If the value of `num` is positive or negative zero, then the result of the method is zero.

Reviewer: ```suggestion
- If the value of `num` is positive/negative zero, then `0.0` is returned.
```

## Example

> The following example shows the application of the `.rint()` method:

Reviewer: ```suggestion
The following example demonstrates the `.rint()` method:
```
